salt/elife-bot/config/opt-elife-bot-letterparser.cfg

```diff
@@ -1,13 +1,13 @@
 [DEFAULT]
 doi_pattern:
 preamble: 
-docker_image: pandoc/core:2.7
+docker_image: pandoc/core:2.9.1.1
 fig_filename_pattern: journalname-{manuscript:0>5}-{id_value}-fig{num}
 video_filename_pattern: journalname-{manuscript:0>5}-{id_value}-video{num}
 
 [elife]
 doi_pattern: 10.7554/eLife.{manuscript:0>5}.{id}
 preamble: <p>In the interests of transparency, eLife publishes the most substantive revision requests and the accompanying author responses.</p>
-docker_image: elifesciences/pandoc:2.7
```

Comment:
if you'd like, put a comment saying `generated with https://alfred.elifesciences.org/job/process/job/process-mirror-pandoc/`

Reply:
I wasn't aware of the process / pipeline - very cool!